automobile/simple-vehicles.md

```diff
@@ -0,0 +1,74 @@
+## Simple Vehicles
+
+### CarSimple
+
+For each model of [Car](#car), a 'simple' PROTO is present too. These simplified
+kinematic PROTO models are not based on a `Robot` node but on a `Solid` node, it is
+therefore not possible to add sensors or control them. They are made to represent
+non-moving parked vehicles or to be moved using a Supervisor because they are much
+faster to simulate than the normal PROTO models.
+
+```
+ PROTO CarSimple {
+   field       SFVec3f    translation             0 0.4 0
+   field       SFRotation rotation                0 1 0 0
+   field       SFColor    color                   0.0 0.25 0.65
+   field       MFColor    recognitionColors       [ 0.0 0.25 0.65, 0.1 0.1 0.1 ]
+   field       MFString   plate                   "textures/plate.jpg"
+   field       SFString   name                    "vehicle"
+   field       SFBool     wheelBoundingObject     FALSE
+ }
+ ```
+
+#### CarSimple Field Summary
+
+- `recognitionColors`: If not empty, this vehicle may be recognized by any Camera
+device with recognition capability (i.e. with a Recognition node).
+- `wheelBoundingObject`: Allows to enable the physical geometry of the wheels.
+
+CarSimple is not an actual PROTO but defines the common structure of all the
+'simple' versions of the cars. The different simple versions of the car PROTO nodes
+represent the different models of [Car](#car):
+
+- the Sport SVR from Range Rover
+- the X5 from BMW
+- the Prius from Toyota
+- the MKZ from Lincoln
+- the C-Zero from Citroen
+
+%figure "Models of cars created using the CarSimple PROTO"
+
+![cars.png](images/cars.png)
+
+%end
+
+### MotorcycleSimple
```

Comment:
Ok I understand know why you use this term.
But since this was not clear to me and to @omichel, we should try to improve it.
I believe you should at least add the list of actual 'motorcycle' (as you did for the car).
and if possible better emphasis that 'MotorcycleSimple' is not an actual PROTO.